django_project/geosight/data/admin/dashboard.py

```diff
@@ -115,10 +116,13 @@ def invalidate_cache(modeladmin, request, queryset):
     queryset.update(version_data=timezone.now())
 
 
-class DashboardAdmin(admin.ModelAdmin):
+class DashboardAdmin(BaseAdminMixin, admin.ModelAdmin):
     """Dashboard admin."""
 
-    list_display = ('slug', 'name', 'creator', 'reference_layer')
+    list_display = (
```

Comment:
Please do like this

list_display = (
        'slug', 'name', 'reference_layer'
    ) + BaseAdminMixin.list_display